### ISSUE TITLE
react-a11y-anchors where link text differs by case & whitespace

### ISSUE THREAD

**Issue by flipchart** [Microsoft/tslint-microsoft-contrib#394]
The rule "Links with the same HREF should have the same link text." in react-a11y-anchors is triggered when the links differ by case, as well as due to irrelevant white space (i.e. indentation). Could this be relaxed to allow differences in case, and to ignore leading and trailing whitespace?

**Comment by JoshuaKGoldberg** [comment]
SGTM. Ideally those could be separate options, individually trigger-able.

**Comment by nadyafebi** [comment]
I'd like to take a shot on this issue if no one is working on it yet :smiley:
